nuxeo-features/nuxeo-user-center/nuxeo-user-profile/src/test/java/org/nuxeo/ecm/user/center/profile/listeners/TestResizeAvatarPictureListener.java

```diff
@@ -64,6 +64,7 @@
 @Deploy("org.nuxeo.ecm.platform.commandline.executor")
 @Deploy("org.nuxeo.ecm.user.center.profile")
 @Deploy("org.nuxeo.ecm.platform.web.common")
+@Deploy("org.nuxeo.ecm.platform.tag")
```

Comment:
I don't get the link between the `nuxeo-user-profile` modifications and the JIRA ticket :)

Reply:
Tricky, there was an exception triggered because the `Picture` schema and docType requires the tag module (for the `NXTag` facet).
Because of this exception, the `bulk` stream processor was not started, which led to a time out when trying to submit a bulk command to remove documents (at test teardown).

Reply:
😱